pkgs/development/ocaml-modules/fix/default.nix

```diff
@@ -19,7 +20,7 @@ stdenv.mkDerivation {
     homepage = "http://gallium.inria.fr/~fpottier/fix/";
     description = "A simple OCaml module for computing the least solution of a system of monotone equations";
     license = licenses.cecill-c;
-    maintainers = [ maintainers.vbgl ];
+    maintainers = with maintainers; [ vbgl ];
     platforms = ocaml.meta.platforms or [];
```

Comment:
Please remove the `platforms` information: it is automatically set by `buildDunePackage`.

Reply:
Done